build_tools/pkgci/external_test_suite/sdxl_scheduled_unet_gpu_rocm.json

```diff
@@ -0,0 +1,32 @@
+{
+    "config_name": "gpu_rocm",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=rocm",
+      "--iree-rocm-target-chip=gfx90a",
+      "--iree-opt-const-eval=false",
+      "--iree-codegen-transform-dialect-library=attention_and_matmul_spec.mlir",
+      "--iree-global-opt-propagate-transposes=true",
+      "--iree-opt-outer-dim-concat=true",
+      "--iree-vm-target-truncate-unsupported-floats",
+      "--iree-llvmgpu-enable-prefetch=true",
+      "--iree-opt-data-tiling=false",
+      "--iree-codegen-gpu-native-math-precision=true",
+      "--iree-codegen-llvmgpu-use-vector-distribution",
+      "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline, util.func(iree-preprocessing-pad-to-intrinsics))"
```

Comment:
Default flags (with XFAIL as needed) and "additional" flags would be good to cover here too